### PR TITLE
Wendy dev api

### PR DESCRIPTION
**Pull request by wj01011**
this is up to date version of home/lesson/quiz content

### CODE REVIEW

File: api/algorithms/src/main/java/com/example/algorithms/quiz/Quiz.java [hunk 1]
```diff
@@ -25,6 +24,7 @@
     private String option3;
     private String posFeedback;
     private String negFeedback;
+=======
```
Comment (MadEste) [inline api/algorithms/src/main/java/com/example/algorithms/quiz/Quiz.java:27]: I think this line is left over from a gitconflict merge?

Comment (MadEste) [inline api/algorithms/src/main/java/com/example/algorithms/quiz/Quiz.java:27]: I think this line is left over from a gitconflict merge?

**Review by MadEste — CHANGES_REQUESTED** [review]
This line should probably be removed

**Comment by wj01011** [comment]
not need to merge this changes.

**Pull request closed without merging**